### PR TITLE
Docs nodetut - Node Tutorial Update

### PR DESCRIPTION
**Pull request by michael-w-williams**
Here is the updated Node tutorial. Made consistent with the Go tutorial.

### CODE REVIEW

File: examples/tutorial/hello/node/README.md [hunk 1]
```diff
@@ -1,83 +1,178 @@
...
+configuration options [here](../../../../docs/operating/options.md). 
+
+## Create your Function 
+
+* Change into the directory where you want to create your function.
+* Create a `func.js` file.
+* Add the following source code and save the file.
+
+```js
+name = "World";
```
Comment (shaunsmith) [inline examples/tutorial/hello/node/README.md]: You may want to keep it even simpler just like the Java tutorial and not pass in a JSON payload?    Here's a non-JSON version https://github.com/fnproject/tutorials/blob/master/ContainerAsFunction/func.js   Of course being able to easily consume JSON is a nice feature of JavaScript.

File: examples/tutorial/hello/node/README.md [hunk 2]
```diff
@@ -1,83 +1,178 @@
...
+```txt
+Deploying hello to app: nodeapp at path: /hello
+Bumped to version 0.0.2
+Building image noreg/hello:0.0.2 
+Updating route /hello using image noreg/hello:0.0.2...
+```
+
+The command creates an app on the server named `nodeapp`. In addition, a route to your function created based on your directory name: `/hello`. The `--local` option allows the application to deploy without a container registry.
+
+## Test your Function on the Server
```
Comment (shaunsmith) [inline examples/tutorial/hello/node/README.md]: It's worth adding a mention and link to the `fn test` support.

**Comment by treeder** [comment]
@michael-w-williams is this PR still valid?

**Comment by michael-w-williams** [comment]
Nope. This isn't needed anymore since we created the language tutorials in the tutorials repo. So I am gonna go ahead and close this.

**Pull request closed without merging**